005_team_3_agent_design/005_team_3_agent_design.tex

```diff
@@ -1 +1,129 @@
-\chapter{Team 3 Agent Design}\label{team_3_agent_design}
\ No newline at end of file
+\chapter{Team 3 Agent Design}\label{team_3_agent_design}
+
+\section{The Agent}\label{the_agent}
+The agent is designed to show different combinations of behaviour depending on three variables we have used to define any agent: stubbornness, \cite{chun_2005},  morality, \cite{waller_2015}, and mood. These three variables, defined in a range of 0 to 100 will change as the agent encounters a series of scenarios, with its personality shifting as events unravel.  The value of the different variables will affect how the agent takes decisions and acts, all of which will be explained in the following pages. \par
+%%ADD THE MAIN CODE!!!
+As the main function from which all the functionality of the agent is bases we have the \texttt{func (a *CustomAgent3) Run()}. The agent follows then next steps:
+\begin{enumerate}
+    \item The agent starts by updating its personality variables at the beginning of each day calling the necessary function. 
+    \item If the agent was just ‘born’ it will learn an initial value for the average food consumed. 
+    \item In the case the agent has changed floors the personality variables are changed depending on the characteristics of the change. 
+    \item In the case the platform is not empty the agent will eat, the quantity depending on \texttt{a.takeFoodCalculation()}.
+    \item Finally the agent will read and send any messages it considers.
+\end{enumerate}
+
+\subsection{Agent knowledge}
```

Comment:
It's worth capitalising all of the headers

Reply:
Done